FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h

```diff
@@ -578,6 +584,40 @@ NS_SWIFT_NAME(Auth)
                    completion:(nullable void (^)(FIRAuthDataResult *_Nullable authResult,
                                                  NSError *_Nullable error))completion;
 
+#if TARGET_OS_IOS || TARGET_OS_TV || TARGET_OS_OSX || TARGET_OS_MACCATALYST
+/**
+ @fn startPasskeySignInWithCompletion:
+ @brief start sign in with passkey retrieving challenge from GCIP and create an assertion request.
+ @param completion Optionally; a block which creates a assertation request.
+
+ @remarks //TODO add possible error codes
```

Comment:
What does this mean? Also add your ldap after TODO.

Reply:
Added ldap.

The reason for this is to follow the current public API reference format which includes a list of possible error codes.  Please see an example [here](https://github.com/firebase/firebase-ios-sdk/blob/passkey-user-change/FirebaseAuth/Sources/Public/FirebaseAuth/FIRAuth.h#L387).  I will fill this after error code bug (b/302700496) for backend is fixed.